### PR TITLE
Refactoring and Improving page.js of designer

### PR DESCRIPTION
**Pull request by Bournvita1998**
Signed-off-by: Bournvita1998 <mohit.kuri@research.iiit.ac.in>

### Description

Refactored and Improved page.js of the designer section.

Fixes #

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

**Review by MauricioFauth — APPROVED** [review]

**Comment by ibennetch** [comment]
Could you please use your real name in the signoff message? You can modify this with `git commit --amend` and push back to your repository (it might require a force-push, which is okay).

File: js/designer/page.js [hunk 1]
```diff
@@ -113,10 +117,11 @@ DesignerPage.loadFirstPage = function (db, callback) {
 DesignerPage.showNewPageTables = function (check) {
     var allTables = $('#id_scroll_tab').find('td input:checkbox');
     allTables.prop('checked', check);
-    for (var tab = 0; tab < allTables.length; tab++) {
+    var allTablesLength = allTables.length;
+    for (var tab = 0; tab < allTablesLength; tab++) {
         var input = allTables[tab];
         if (input.value) {
-            var element = document.getElementById(input.value);
+            var element = $('#' + input.value);
```
Comment (devenbansod) [inline js/designer/page.js]: @Bournvita1998 even this won't work, as we discussed in #15338

File: js/designer/page.js [hunk 2]
```diff
@@ -132,13 +137,14 @@ DesignerPage.loadHtmlForPage = function (pageId) {
     DesignerPage.loadPageObjects(pageId, function (page, tblCords) {
         $('#name-panel').find('#page_name').text(page.pageDescr);
         DesignerMove.markSaved();
-        for (var t = 0; t < tblCords.length; t++) {
+        var tblCordsLength = tblCords.length;
+        for (var t = 0; t < tblCordsLength; t++) {
             var tbId = db + '.' + tblCords[t].tableName;
-            var table = document.getElementById(tbId);
+            var table = $('#' + tbId);
```
Comment (devenbansod) [inline js/designer/page.js]: @Bournvita1998 same as above

File: js/designer/page.js [hunk 3]
```diff
@@ -132,13 +137,14 @@ DesignerPage.loadHtmlForPage = function (pageId) {
...
             table.style.top = tblCords[t].y + 'px';
             table.style.left = tblCords[t].x + 'px';
 
-            var checkbox = document.getElementById('check_vis_' + tbId);
+            var checkbox = $('#check_vis_' + tbId);
```
Comment (devenbansod) [inline js/designer/page.js]: @Bournvita1998 same as above

**Review by devenbansod — CHANGES_REQUESTED** [review]
Please make sure you test your changes (ideally even before you commit, but definitely before you submit the PR)

**Pull request closed without merging**